spec/Section 6 -- Execution.md

```diff
@@ -121,16 +121,16 @@ respectively.
 ### Query
 
 If the operation is a query, the result of the operation is the result of
-executing the operation’s top level selection set with the query root operation
-type.
+executing the operation’s top level _selection set_ with the query root
+operation type.
 
 An initial value may be provided when executing a query operation.
 
 ExecuteQuery(query, schema, variableValues, initialValue):
 
 - Let {queryType} be the root Query type in {schema}.
 - Assert: {queryType} is an Object type.
-- Let {selectionSet} be the top level Selection Set in {query}.
+- Let {selectionSet} be the top level selection set in {query}.
```

Comment:
Needs to be `_selection set_`?

Reply:
did this get addressed?

Reply:
It was concluded that it was unnecessary since its referenced outside of the algorithm in the previous paragraph.